src/pages/invite/[uuid].svelte

```diff
@@ -16,7 +16,9 @@ onMount(() => {
   {#if $loading}
     <p class="m-0-auto">Processing invitation...</p>
   {:else if $error.key === 'ErrorProcessingAuthInviteCode'}
-    <p class="m-0-auto">That invite code is invalid or too old</p>
+    <p class="m-0-auto">That invite code is invalid</p>
+  {:else if $error.key === 'ErrorInviteExpired'}
+    <p class="m-0-auto">That invite code has expired</p>
```

Comment:
Would it work to add it [here](https://github.com/silinternational/cover-api/blob/develop/application/locales/errors.en.yaml)?

Reply:
I suppose I would be able to do this then
```suggestion
    <p class="m-0-auto">    <p class="m-0-auto">{$error.message}</p>
```

Reply:
I sort of expected you wouldn't need to handle it at all. Isn't the `message` already used elsewhere?